public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -719,40 +720,75 @@ const MuteTimings: FC<{ timings: string[]; alertManagerSourceName: string }> = (
   const groupWait = timingOptions.group_wait;
   const groupInterval = timingOptions.group_interval;
+  const repeatInterval = timingOptions.repeat_interval;
 
   // we don't have any timing options to show – we're inheriting everything from the parent
   // and those show up in a separate "inherited properties" component
-  if (!groupWait && !groupInterval) {
+  if (!groupWait && !groupInterval && !repeatInterval) {
     return null;
   }
 
+  const metaOptions: ReactNode[] = [];
+
+  if (groupWait) {
+    metaOptions.push(
+      <Tooltip
+        placement="top"
+        content="How long to initially wait to send a notification for a group of alert instances."
+      >
+        <span>
+          <Trans i18nKey="alerting.policies.metadata.timingOptions.groupWait">
+            Wait <PrimaryText content={groupWait} /> to group instances
+          </Trans>
+        </span>
+      </Tooltip>
+    );
+  }
+
+  if (groupInterval) {
+    metaOptions.push(
+      <Tooltip
+        placement="top"
+        content="How long to wait before sending a notification about new alerts that are added to a group of alerts for which an initial notification has already been sent."
+      >
+        <span>
+          <Trans i18nKey="alerting.policies.metadata.timingOptions.groupInterval">
+            <PrimaryText content={groupInterval} /> before sending updates
+          </Trans>
+        </span>
+      </Tooltip>
+    );
+  }
+
+  if (repeatInterval) {
+    metaOptions.push(
+      <Tooltip
+        placement="top"
+        content="How often notifications are sent if the group of alerts has not changed since the last notification."
+      >
+        <span>
+          <Trans i18nKey="alerting.policies.metadata.timingOptions.repeatInterval">
+            repeated every <PrimaryText content={repeatInterval} />
+          </Trans>
+        </span>
+      </Tooltip>
+    );
+  }
+
   return (
     <MetaText icon="hourglass" data-testid="timing-options">
-      <span>Wait</span>
-      {groupWait && (
-        <Tooltip
-          placement="top"
-          content="How long to initially wait to send a notification for a group of alert instances."
-        >
-          <span>
-            <Text color="primary">{groupWait}</Text> to group instances
-            {groupWait && groupInterval && ','}
-          </span>
-        </Tooltip>
-      )}
-      {groupInterval && (
-        <Tooltip
-          placement="top"
-          content="How long to wait before sending a notification about new alerts that are added to a group of alerts for which an initial notification has already been sent."
-        >
-          <span>
-            <Text color="primary">{groupInterval}</Text> before sending updates
-          </span>
-        </Tooltip>
-      )}
+      {metaOptions.map((meta, index) => (
+        <span key={uniqueId()}>
+          {meta}
+          {index < metaOptions.length - 1 && ', '}
+        </span>
+      ))}
```

Comment:
In terms of translations, we might be making it harder to make this make sense in other languages. It's because previously we had "wait" at the start separately, and the assumption is that every part that's joined with a comma makes sense with regards to "wait". Commas might need to go in different places as well? I'm not sure

It might be clearer to split the 3 potential phrases more, and put fullstops/something else in between?

```
Wait 30s to group instances | Wait 5m before sending updates | Repeated every 4h
```